lib/charms/mongodb_libs/v0/machine_helpers.py

```diff
@@ -148,6 +148,9 @@ def generate_service_args(auth: bool, machine_ip: str, replset: str) -> str:
         # part of replicaset
         "--replSet",
         f"{replset}",
+        # logging should be sent to /data/db as to ensure persistence on machine removal.
+        # logappend ensures we have all logs instead of rotating through logs
+        "--logpath=/data/db/mongodb.log --logappend",
```

Comment:
I think it is important to do some rotation after some time, by using something like [db.adminCommand( { logRotate : "server" } )
](https://www.mongodb.com/docs/manual/tutorial/rotate-log-files/#rotate-the-log-file-1) at some point in the code and remobing old log files, to not fulfill the entire storage. This can be done in another PR.